packages/deployments/contracts/contracts/nomad-xapps/contracts/promise-router/PromiseRouter.sol

```diff
@@ -26,13 +30,22 @@ contract PromiseRouter is Version0, Router {
   // ========== Custom Errors ===========
 
   error PromiseRouter__onlyConnext_notConnext();
-  error PromiseRouter__send_calldataEmpty();
-  error PromiseRouter__send_callbackAddressEmpty();
+  error PromiseRouter__send_returndataEmpty();
+  error PromiseRouter__send_callbackAddressNotContract();
+  error PromiseRouter__process_InvalidMessage();
+  error PromiseRouter__process_notApprovedRelayer();
+  error PromiseRouter__process_insufficientCallbackFee();
+  error PromiseRouter__process_notContractCallback();
+  error PromiseRouter__bumpCallbackFee_valueIsZero();
 
   // ============ Public Storage ============
 
   IConnext public connext;
 
+  // callback messages transferId => message
+  mapping(bytes32 => bytes) public promiseMessages;
+  mapping(bytes32 => uint256) public callbackFees;
```

Comment:
all of these stored items need natspec documentation on them

Reply:
Added